modules/moduleadmin/code/controller.ext.php

```
@@ -247,37 +247,37 @@ static function getIsShowModuleInfo() {
 
     static function getModuleInfoName() {
         global $controller;
-        $info = ui_module::GetModuleXMLTags($controller->GetControllerRequest('URL', 'showinfo'));

```

BTW mate... Although this fix suppresses the errors appearing when full PHP debug mode is enabled this doesn't fix it as the ?showinfo parameter is parsed and used when the module name  is clicked a new screen then displays information about the module of which name is passed in the showinfo parameter so I'll deal with this and the fix just requires an isset() wrapper to check if the user has clicked... I'll fix this now and commit the fix for this module. What happens with your current fix for this is that whatever module name is clicked the 'Module information' screen always shows information of the Module Admin module and not the individual module :)